### PR TITLE
updates of repos, licenses and urls

### PR DESCRIPTION
**Pull request by Trilarion**
I have some updates of repo-urls, licenses and home-urls for the games.

I there are questions about some of these updates, please ask.

I also normalized the endings of the Github repos (usually they ended without a slash, sometimes they did, I removed the trailing slashes but did not update the updated field because of that).

### CODE REVIEW

File: games/c.yaml [hunk 1]
```diff
@@ -889,25 +889,27 @@
...
   license:
   - GPL3
-  development: halted
+  development: active
   originals:
   - Kula World
-  repo: http://sourceforge.net/projects/cubosphere/
+  repo:
+  - https://github.com/cubosphere/cubosphere-code
+  - https://sourceforge.net/p/cubosphere/code/
```
Comment (cxong) [inline games/c.yaml]: can't have multiple repos

File: games/g.yaml [hunk 2]
```diff
@@ -215,8 +215,9 @@
   originals:
   - Minecraft
   status: playable
+  repo: https://github.com/Gnomescroll/Gnomescroll
   type: remake
-  updated: 2014-08-20
+  updated: 2019-11-20
   url: https://github.com/Gnomescroll/Gnomescroll
```
Comment (cxong) [inline games/g.yaml]:
```suggestion
```

File: games/g.yaml [hunk 3]
```diff
@@ -254,7 +255,9 @@
   - GoldenEye 007
   type: remake
   repo: https://github.com/goldeneye-source/ges-code
-  url: https://www.moddb.com/mods/goldeneye-source
+  url:
+  - http://geshl2.com/
+  - https://www.moddb.com/mods/goldeneye-source
```
Comment (cxong) [inline games/g.yaml]: cannot have multiple URLs

File: games/n.yaml [hunk 4]
```diff
@@ -76,15 +76,16 @@
   - http://i.imgur.com/4sJM8Ji.jpg
   - http://i.imgur.com/fzcotpu.jpg
   lang:
+  - C
   - C++
```
Comment (cxong) [inline games/n.yaml]:
```suggestion
```

File: games/o.yaml [hunk 5]
```diff
@@ -1442,14 +1443,14 @@
   lang:
   - C#
   license:
-  - GPL3
+  - LGPL3
   development: active
   originals:
   - 'Command & Conquer: Generals'
   repo: https://github.com/OpenSAGE/OpenSAGE
   type: clone
```
Comment (cxong) [inline games/o.yaml]:
```suggestion
  type: remake
```

File: games/p.yaml [hunk 6]
```diff
@@ -371,7 +371,7 @@
   - Civilization II
   repo: https://github.com/freeciv/freeciv-web
   type: clone
-  url: http://play.freeciv.org/
+  updated: 2019-11-20
```
Comment (cxong) [inline games/p.yaml]:
```suggestion
  url: https://www.freecivweb.org/
  updated: 2019-11-20
```

File: games/c.yaml (hunk 1 above)
Comment (Trilarion) [reply games/c.yaml]: will remove the old one

File: games/g.yaml (hunk 3 above)
Comment (Trilarion) [reply games/g.yaml]: will remove one

**Comment by Trilarion** [comment]
Requested changes are implemented.

**Pull request merged**